app/views/issues/update.json.erb

```diff
@@ -1,7 +1,7 @@
 {
   "success": <%= @modified %>,
   <% if tag = @issue.tags.first %>
-  "issue_link": "<%= j render('issues/issue.html.erb', issue: @issue) %>",
+  "issue_link": "<%= j render(partial: 'issues/issue', locals: { issue: @issue }, formats: 'html') %>",
```

Comment:
@sean-yeoh fyi you can replace this syntax (`{locals: { issue: @issue } }` with `object: @issue`)

Reply:
Ehh.. cool, it works. I'll do that actually!